<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>926</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QEntropyGraph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>907</width>
      <height>272</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="layoutPath">
       <item>
        <widget class="QLabel" name="labelPath">
         <property name="text">
          <string>Path:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="editPath">
         <property name="text">
          <string>/Users/Duncan/Projects/EntropyGraph/EntropyGraph/bin/Debug/EntropyGraph.exe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnRandomGraph">
         <property name="text">
          <string>Random Graph</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnGraphEntropy">
         <property name="text">
          <string>Graph Entropy</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutBlockSize">
         <item>
          <widget class="QLabel" name="labelBlockSize">
           <property name="text">
            <string>Block Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editBlockSize">
           <property name="text">
            <string>256</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutPointCount">
         <item>
          <widget class="QLabel" name="labelPointCount">
           <property name="text">
            <string>Point Count:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editPointCount">
           <property name="text">
            <string>300</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutColor">
         <item>
          <widget class="QLabel" name="labelColor">
           <property name="text">
            <string>Color:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="editColor">
           <property name="font">
            <font>
             <family>Courier New</family>
            </font>
           </property>
           <property name="text">
            <string>#00FF00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
